tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+# conftest file for pytest
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+
+from crystal_toolkit.core.scene import Cylinders, Lines, Spheres, Surface
+
+
+@pytest.fixture(scope="session")
+def test_files():
+    return Path(__file__).parent / "test_files"
+
+
+@pytest.fixture(scope="session")
```

Comment:
If only used in `tests/asy_test/test_asymptote.py`, maybe we should move the fixture there?

Reply:
Makes sense I will move it.